### DISCUSSION TITLE
About JCache changes in 15.x releases

### DISCUSSION THREAD

**Discussion by Kiiv** [infinispan/infinispan#11323] [Q&A]
Hi,

I was planning to use Infinispan in my project  through the JCache API. 
I've since read the [blog post](https://infinispan.org/blog/2023/07/05/infinispan-15dev02) about changes in versions 15.x and I'm wondering if I still can rely serenely on JCache API or if I need to use a different approach ?

From the blog post I understand that if I don't use any of the classes inside javax.cache.annotation.* package, I'm good. 

> We understand the need to have an implementation agnostic caching API, so we are going to continue supporting our JCache provider, but we will drop those parts which require Java EE, most notably the javax.cache.annotation package.

But I can't find any infinispan-jcache 15.x version on maven repository so I'm wondering if my guess is correct ? :/

Regards,
Clément.

**Comment by tristantarrant** [comment]
The JCache module removal in 15.0 is temporary and it will be back at some point in the "reduced" implementation described in the blog post.

**Reply by tristantarrant** [reply]
Addendum: JCache is effectively abandonware at this point and cannot ever come back, because of IP implications. A new caching API may appear as part of Jakarta EE, but not soon.

**Reply by Kiiv** [reply]
Ok, so I understand that starting a new implementation based on JCache is not a good bet ;)

Thanks for your quick answer !
